components/ProjectSideBar/projectSideBar.js

```diff
@@ -58,12 +62,12 @@ export const ProjectSideBar = ({ campaign, onClick, isOwner, setIsPledge }) => {
       <BoxContainer>
         <Box>
           <Subtitle>Revenue</Subtitle>
-          <Value>{currentAmount[0].amount}USDT</Value>
+          <Value>{currentAmount[0].amount} USDT</Value>
```

Comment:
NIT: As we talked with @lucasgalarce the token symbol should be dynamic. We can improve it in the future